core/server/adapters/scheduling/post-scheduling/index.js

```diff
@@ -42,12 +79,16 @@ _private.loadClient = function loadClient() {
  */
 _private.loadScheduledPosts = function () {
     // TODO: make this version aware?
```

Comment:
Did you give a thought about making this piece in some way "configurable" in sense of supported/used version? For example, we can easily use v3 here instead of v2, and we need somehow to give this clue to the scheduler from within Ghost

Reply:
@gargol Not too much, but think we should start using the concept of `availableVersions` here to pick from? Was also thinking if we should add utils to a version to give the URL for publish from scheduler than calculating it in scheduling itself. Another area had in mind to come back to after all functionality test. :)

Reply:
@gargol Updated this to be version aware in its own way by picking the default api version provided, which it expects to have certain format which is true for both `v3` and `v2` atm. I think its a good solution for now, we'll always just need to ensure to update the logic to get schedules resources for default API version. Lmk if you have any ideas :)

Reply:
yeah, think this is most sensible for now while we don't have many differences between v2 and v3 :+1: